tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-docker-release-pipeline.yml

```diff
@@ -0,0 +1,72 @@
+parameters:
+  - name: commit_sha1
+    displayName: Commit Sha1
+    type: string
+    default: master
+  - name: image_tag
+    displayName: Image Tag
+    type: string
+    default: latest
+
+stages:
+- stage: Python_Packaging
```

Comment:
the stage name isn't that fitting. since there aren't multiple stages, can probably specify jobs at the top level.